content/troubleshooting/java/installation/exec-helper-troubleshooting.md

```diff
@@ -2,39 +2,45 @@
 
 Newer systems which use systemd, including EL7 and Ubuntu 16.04, store Syslog messages in the systemd journal. Use `journalctl` to view the messages (i.e., `journalctl -t Contrast`). Traditional init.v systems, including EL6 and Ubuntu 14.04, store Syslog messages in the file */var/log/messages* by default.
 
-If there are no logs in either of these storage locations, verify Syslog configuration `/etc/syslog.conf`; the host may be configured to drop the message types or log level used by the Contrast Java agent Exec Helper.
+If there aren't any logs in either of these storage locations, verify Syslog configuration `/etc/syslog.conf`; the host may be configured to drop the message types or log level used by the Exec Helper.
 
-## Issue: I removed the libcontrast_universal_installer.so libraries, and now my system errors on every command I run. 
+## Issue: I removed the libcontrast_universal_installer.so libraries, and now my system errors on every command I run 
 
-The host emits these errors because the \*nix system has been globally configured to load the *libcontrast_universal_installer.so* library for every binary on the system to determine if it should inject the Java agent into it. Since the library was manually removed, the global configuration of dynamic loader is still referencing it and is failing/warning when it can't find the file.
+The host emits these errors because the *\*nix* system has been globally configured to load the *libcontrast_universal_installer.so* library for every binary on the system to determine if it should inject the Java agent into it. Even if you manually removed the library, the global configuration of dynamic loader is still referencing it and is failing/warning when it can't find the file.
+
+### Linux 
 
 On a Linux system, the shell command is still running, even though you see errors about failing to load the *contrast.so* library. Open up the */etc/ld.so.preload* file as root and remove the entry for the contrast libraries.
 
-On a Solaris system, the ld.so process will fail to execute the desired binary if the ld preload library doesn't exist. This causes all commands to be broken. To recover if you still have a shell prompt, use the `LD_NOCONFIG` environment variable to temporarily disable the custom dynamic loader config and remove the config itself. In Solaris, there are two configs that must be removed - one for 32bit and one for 64bit.
+### Solaris 
+
+On a Solaris system, the ld.so process will fail to execute the desired binary if the *ld preload* library doesn't exist. This causes all commands to be broken. 
+
+If you need to recover, and still have a shell prompt, use the `LD_NOCONFIG` environment variable to temporarily disable the custom dynamic loader `config` setting and remove the *config* file itself. In Solaris, there are two `config` settings that must be removed - one for 32-bit and one for 64-bit.
```

Comment:
@gilday I tried to make some distinction between the uses/meanings of 'config'. Please adjust as needed.

Reply:
tbh I'm not a Solaris user so this is a bit over my head. @seschis can you please take a look at @kaitlynmortimer's edits?

Reply:
@kaitlynmortimer I just realized Shane is on vacation. If you're good with this PR, let's merge it into that linux-pkgs branch